src/file/file.spec.ts

```diff
@@ -0,0 +1,135 @@
+import * as chai from 'chai';
```

Comment:
I do not see here testing errors thrown. The generic controller's spec takes care of most of these, see if it helps you here. 
Also, you might need to add error classes or some kind of indicator in case things gone wrong and the user won't know what.